console/console-init/ui/stories/AddressList.stories.tsx

```diff
@@ -1,10 +1,13 @@
 import * as React from "react";
 import { MemoryRouter } from "react-router";
 import { storiesOf } from "@storybook/react";
-import { AddressList, IAddress } from "../src/Components/AddressList";
+import { AddressList, IAddress } from "../src/Components/AddressSpace/AddressList";
 import { action } from "@storybook/addon-actions";
+import { boolean, select, withKnobs } from "@storybook/addon-knobs";
+import {AddressListFilter} from "src/Components/AddressSpace/AddressListFilter";
```

Comment:
Be consistent in the imports, here there is `../src` and `src` used at the same time that I guess are both pointing to the same thing?

Reply:
Yeah, I think it happened because of auto import by the editor.